FirebaseDynamicLinks/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # 11.8.0
-- [added] Deprecate FirebaseDynamicLinks CocoaPod in advance of August 25, 2025 Dynamic
-Links service shutdown. See https://firebase.google.com/support/dynamic-links-faq.
+- [deprecated] Dynamic Links is deprecated. For information about timelines and
+  alternatives, see the https://firebase.google.com/support/dynamic-links-faq.
```

Comment:
Also, line 6?

Reply:
10.27.0 deprecated the APIs. 11.8.0 adds the deprecation notice to the CocoaPod.

Reply:
Changed the tag to deprecated on the 10.27.0 release note. I'm ok with the text remaining the same.